CryptKeeperZK/crypt-keeper-extension
# feat: identity page

## Explanation

This PR adds identity page to show user full identity metadata.

Details are below:
- [x] Add identity page
- [x] Fix firefox manifest v2
- [x] Minor refactoring for identity list item
- [x] Add delete and update identity actions
- [x] Add e2e test for identity page

## More Information

Closes #683 

## Screenshots/Screencaps

Extended mode:
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/36d326c9-0ad8-4581-b531-c33cd382b5b7)
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/8da37aa4-061a-4b7f-baa9-d3f5ad2c01a3)

Popup mode:
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/ba2af657-c932-4f8d-bafa-ba4faf83c522)
![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/c9ba2e12-29f9-417a-b92b-ecbcad133a63)


## Manual Testing Steps

1. Go to home
2. Open identity view
3. See identity metadata

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
